cmd/internal/cli/starter_linux.go

```diff
@@ -695,6 +873,9 @@ func execStarter(cobraCmd *cobra.Command, image string, args []string, name stri
 		sylog.Infof("Setting --net (required by --network-args)")
 		NetNamespace = true
 	}
+	// unprivileged installation could not use fakeroot
+	// network because it requires a setuid installation
+	// so we fallback to none
```

Comment:
This comment should be left where it was, above the check starting `if !starter.IsSuidInstall()` below.